src/main/java/dan200/computercraft/shared/peripheral/monitor/ClientMonitor.java

```diff
@@ -50,6 +56,29 @@ public boolean createBuffer( MonitorRenderer renderer )
     {
         switch( renderer )
         {
+            case TBO:
+            {
+                if( tboBuffer != 0 ) return false;
+
+                deleteBuffers();
+
+                tboBuffer = GlStateManager.genBuffers();
+                GlStateManager.bindBuffer( GL31.GL_TEXTURE_BUFFER, tboBuffer );
+                // TODO: DYNAMIC? Probably not, but worth profiling. After all, it only needs to be fast on /my/
+                //  graphics card.
+                GL15.glBufferData( GL31.GL_TEXTURE_BUFFER, 0, GL15.GL_STATIC_DRAW );
```

Comment:
Dynamic! If a buffer ever changes at all, make it dynamic. The real question here is if we want it to be `GL_DYNAMIC_DRAW` or `GL_STREAM_DRAW`. The latter is to be used for buffers that are updated very frequently.

Though this still needs profiling, these flags are hints to the driver, not commands.

Reply:
What constitutes as "very frequently" here? I guess my thinking is that a lot of monitors change rarely (at most once every 10 seconds, if ever). But given that's not the slow path right now, maybe not worth optimising for.

Reply:
Well, it's called `STREAM_DRAW` because it's meant for buffers that are streamed. That is, it's usually used for meshes that update almost every single frame (e.g. character animations). But, as I said, it's not very well-defined, as it's entirely up to the driver.